server/src/main/java/org/elasticsearch/search/aggregations/support/AggregationInspectionHelper.java

```diff
@@ -104,7 +104,7 @@ public static boolean hasValue(InternalFilter agg) {
         return agg.getDocCount() > 0;
     }
 
-    public static boolean hasValue(InternalGeoHashGrid agg) {
+    public static boolean hasValue(GeoHashGrid agg) {
```

Comment:
Any reason this is not `public static boolean hasValue(InternalGeoGrid<?> agg) {`?

Reply:
I left it as the implementation since that was its only use-case. but as we add the next aggregation, that will change. so I will update here to reflect